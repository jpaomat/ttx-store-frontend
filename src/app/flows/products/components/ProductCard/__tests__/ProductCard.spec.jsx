import Renderer from 'react-test-renderer/shallow';
const { ProductCard } = require('../ProductCard');
const { defaultMock } = require('./mocks');

const renderer = new Renderer();

describe('Techtronix component', () => {
	it('should render component', () => {
		const component = renderer.render(<ProductCard {...defaultMock}/>);
		expect(component).toMatchSnapshot();
	});
});