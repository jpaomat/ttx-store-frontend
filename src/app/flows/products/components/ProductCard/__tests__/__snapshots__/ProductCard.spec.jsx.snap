// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Techtronix component should render component 1`] = `
<div
  className="card"
>
  <img
    className="card__image"
    src="http:image-hp-test.com"
  />
  <div
    className="card__content"
  >
    <p
      className="card__content--brand"
    >
      Hp
    </p>
    <p
      className="card__content--model"
    >
      test model
    </p>
    <PriceComponent
      price="110"
    />
  </div>
  <ButtonApp
    disabledBtn={false}
    onClick={[Function]}
    size="sm"
  >
    <ButtonContent
      price="110"
    />
  </ButtonApp>
</div>
`;
